internal/provisioner/cluster_register.go

```diff
@@ -86,7 +68,7 @@ func (cr *ClusterRegister) clusterRegister(s3StateStore string) error {
 		return errors.Wrap(err, "failed to commit to repo")
 	}
 
-	if err = cr.gitClient.Push(logger); err != nil {
+	if err = cr.gitClient.Push("feat-CLD-5708", logger); err != nil {
```

Comment:
It should be `main`.  I changed it back for testing purposes and forgot I'll fix it. We need to change it to test only I don't think it needs to be dynamic.

Reply:
tbh, it'll be better to have it configurable to avoid changing t in different places of the code in cases of test in a specific branch. I'll review it.

Reply:
fixed it